app/controllers/coronavirus/pages_controller.rb

```diff
@@ -11,7 +10,7 @@ def index
     end
 
     def show
-      page
+      return not_found if page.blank?
```

Comment:
As mentioned on an earlier comment this can be avoided with the bang variant of `find_by` (well if I remember correctly)